Automattic/jetpack
# Remove $connected_destination_account_id from call

Fixes #32092 

## Proposed changes:

* A recent refactor (D115524-code#C8427620OL522) in WPCOM code changed the call signature for `generate_default_products` by removing `connected_account_id`
* This removes the same argument in the function call

## Testing instructions:






- Use a site where you've never added a Donations Block or a create new site
- Use store sandbox (PCYsg-IA-p2)
- Connect a Stripe account  following these instructions (PCYsg-lW4-p2)
- On a new or existing page, add a new block and choose "Donation Form"
- Pause a second and then press "Update" to save the page
- Using wpsh (PCYsg-4Ki-p2), run `gpr SELECT post_id, meta_key, meta_value FROM wp_[site_id]_postmeta WHERE meta_key IN ('jetpack_memberships_acct_id', 'jetpack_memberships_is_editable') ORDER BY post_id, meta_key` where `site_id` is the site ID
- verify that `jetpack_memberships_is_editable` returns either 0 or 1 values and no `post_id` has the `jetpack_memberships_acct_id` in the `jetpack_memberships_is_editable` value
